src/commcare_cloud/commands/inventory_lookup/getinventory.py

```diff
@@ -29,6 +31,10 @@ def get_server_address(environment, group, exit=sys.exit):
         username += "@"
     else:
         username = ""
+
+    if re.match(r'(\d+\.?){4}', group):
```

Comment:
Is this for an ip address? may be worth a comment for this (or some constant with reasonable name)